### PR TITLE
Minor code improvements

### PR DESCRIPTION
**Pull request by alexgnt**
- passing string parameters by const ref. in iomusxml and optimization
of convert functions
- removed not needed call of AlignVertically for system aligner

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]

**Comment by alexgnt** [comment]
@lpugin , please have a look at this one.

**Comment by craigsapp** [comment]
Looks great to me, although @lpugin can comment about the use of static variables, which he was complaining about in another recent pull request.

File: src/iomusxml.cpp [hunk 1]
```diff
@@ -3156,121 +3151,99 @@ void MusicXmlInput::ReadMusicXmlPrint(pugi::xml_node node, Section *section)
...
+        { "natural-down", ACCIDENTAL_WRITTEN_nd },
+        { "natural-up", ACCIDENTAL_WRITTEN_nu },
+        { "flat-down", ACCIDENTAL_WRITTEN_fd },
+        { "flat-up", ACCIDENTAL_WRITTEN_fu },
+        { "triple-sharp", ACCIDENTAL_WRITTEN_ts },
+        { "triple-flat", ACCIDENTAL_WRITTEN_tf }
+    };
+
+    const auto result = Accidental2Accid.find(value);
+    if (result != Accidental2Accid.end())
```
Comment (lpugin) [inline src/iomusxml.cpp:3180]: Put on one line or add {} (Same elsewhere)

**Review by lpugin — CHANGES_REQUESTED** [review]
Looks good. Please run clang-format.

File: src/iomusxml.cpp (hunk 1 above)
Comment (alexgnt) [reply src/iomusxml.cpp]: Done

**Comment by alexgnt** [comment]
> Looks good. Please run clang-format.

I've fixed only if () {} formatting, since clang, tries to put initialization list on the line value-by-value. What makes it harder to read it, as for me.

**Comment by lpugin** [comment]
> > Looks good. Please run clang-format.
>
> I've fixed only if () {} formatting, since clang, tries to put initialization list on the line value-by-value. What makes it harder to read it, as for me.

Yes, I know, clang formatting can be annoying in some cases. You can try to see if there are some options we can change in our clang-format configuration. Run it anyway.

**Comment by alexgnt** [comment]
> > > Looks good. Please run clang-format.
> > 
> > 
> > I've fixed only if () {} formatting, since clang, tries to put initialization list on the line value-by-value. What makes it harder to read it, as for me.
> 
> Yes, I know, clang formatting can be annoying in some cases. You can try to see if there are some options we can change in our clang-format configuration. Run it anyway.

This has been done.

**Comment by lpugin** [comment]
OK, I looked at it. You can actually do
```
    // clang-format off
    static const std::map<float, data_ACCIDENTAL_GESTURAL> Alter2Accid {
        { -2, ACCIDENTAL_GESTURAL_ff },
        { -1.5, ACCIDENTAL_GESTURAL_fd },
        { -1, ACCIDENTAL_GESTURAL_f },
        { -0.5, ACCIDENTAL_GESTURAL_fu },
        { 0, ACCIDENTAL_GESTURAL_n },
        { 0.5, ACCIDENTAL_GESTURAL_sd },
        { 1, ACCIDENTAL_GESTURAL_s },
        { 1.5, ACCIDENTAL_GESTURAL_su },
        { 2, ACCIDENTAL_GESTURAL_ss }
    };
    // clang-format on
```
or
```
    static const std::map<float, data_ACCIDENTAL_GESTURAL> Alter2Accid{
        { -2, ACCIDENTAL_GESTURAL_ff }, //
        { -1.5, ACCIDENTAL_GESTURAL_fd }, //
        { -1, ACCIDENTAL_GESTURAL_f }, //
        { -0.5, ACCIDENTAL_GESTURAL_fu }, //
        { 0, ACCIDENTAL_GESTURAL_n }, //
        { 0.5, ACCIDENTAL_GESTURAL_sd }, //
        { 1, ACCIDENTAL_GESTURAL_s }, //
        { 1.5, ACCIDENTAL_GESTURAL_su }, //
        { 2, ACCIDENTAL_GESTURAL_ss } //
    };
```

**Comment by lpugin** [comment]
The second option is probably better because other formatting steps (e.g., tabulations) are still applied. Only line breaks formatting is suppressed.

**Comment by alexgnt** [comment]
> The second option is probably better because other formatting steps (e.g., tabulations) are still applied. Only line breaks formatting is suppressed.

Done.

**Review by lpugin — APPROVED** [review]

**Pull request merged**